ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -5734,3 +5734,42 @@ def test_torch_special_and(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# bitwise_xor
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="bitwise_xor",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=st.one_of(st.just(("bool",)), helpers.get_dtypes("integer")),
+        num_arrays=2,
+        min_value=-1e04,
+        max_value=1e04,
```

Comment:
Hey @jatinS-dev! I think we should get the tests passing without setting a limit to these values since `torch` natively accepts integers beyond these limits. See [here](https://pytorch.org/docs/stable/generated/torch.bitwise_xor.html#torch.bitwise_xor)

Reply:
Hi @hmahmood24 I have resolved it now, you can check